src/wpool_utils.erl

```diff
@@ -15,8 +15,6 @@
 %%% @doc Common functions for wpool_process and other modules.
 -module(wpool_utils).
 
--author('ferigis@gmail.com').
-
```

Comment:
Elvis... lemme know if you prefer to turn this into a `@author` in the comment section above...